metals/src/main/scala/scala/meta/internal/metals/ServerCommands.scala

```diff
@@ -161,9 +161,8 @@ object ServerCommands {
        |If symbol under cursor is invalid or does not override anything then command is ignored.
        |
        |Note: document in json argument must be absolute path.
-       |
-       |Arguments:
-       |```json
+       |""".stripMargin,
+    """|```json
```

Comment:
So it looks like this one is still a bit screwy with this change.

<img width="900" alt="Screenshot 2020-04-13 at 10 49 55" src="https://user-images.githubusercontent.com/13974112/79107441-aa982f00-7d74-11ea-80ae-dd3d82f5846f.png">

Reply:
probably because it is a multi line string. 
What is the sbt command to generate the doc ?

Reply:
You can find instructions on how to generate the docs and view them here https://scalameta.org/metals/docs/contributors/updating-website.html